ee/clickhouse/models/event.py

```diff
@@ -111,7 +111,7 @@ def bulk_create_events(events: List[Dict[str, Any]], person_mapping: Optional[Di
                 person_id = person.uuid
             except Person.DoesNotExist:
                 person_properties = {}
-                person_id = cast(uuid.UUID, "00000000-0000-0000-0000-000000000000")
+                person_id = uuid.uuid4()
```

Comment:
This basically implies that there's funnel tests where we have distinctIDs in events that don't match any person, i.e. no persons were created for these events.

I feel like that's a test bug to be fixed in tests instead? Although, depends on how many tests are like these. What did you find?

Reply:
Ugh, the merge branch muddied the commit history so I can't check if there was a specific failing test but see response to your last comment for more context too